server/server.js

```diff
@@ -71,6 +71,20 @@ app.get(`${apiBaseUrl}/participants`,
     return res.json({ data: result });
   }));
 
+// Engage participant
+app.post(`${apiBaseUrl}/engage-participant`,
```

Comment:
I'm wondering if we actually want a generic endpoint here - like POST to /employer-actions since we're already handling the status in the request body? Then we would just need to expand this endpoint to handle more options vs. creating bespoke endpoints for each status

Thoughts?

Reply:
Totally makes sense, I just turn it into a generic actions endpoint for now